subprojects/core/src/test/groovy/org/gradle/caching/internal/controller/DefaultBuildCacheControllerTest.groovy

```diff
@@ -220,41 +213,6 @@ class DefaultBuildCacheControllerTest extends Specification {
         1 * remote.close()
     }
 
-    def "emits operation when disabling service from recoverable errors"() {
-        given:
-        MAX_ERRORS * remote.store(key, _) >> { throw new BuildCacheException("Remote") }
-
-        when:
-        def controller = getController()
-        MAX_ERRORS.times {
-            controller.store(storeCommand)
-        }
-
-        then:
-        def ops = operations.log.all(BuildCacheDisableServiceBuildOperationType)
-        ops.size() == 1
-        with(ops[0].descriptor.details, BuildCacheDisableServiceBuildOperationType.Details) {
-            message == "3 recoverable errors were encountered"
-            reason == TOO_MANY_RECOVERABLE_ERRORS
-        }
-    }
-
-    def "emits operation when disabling service from non-recoverable errors"() {
-        given:
-        1 * remote.store(key, _) >> { throw new IOException("Remote") }
-
-        when:
-        controller.store(storeCommand)
-
-        then:
-        def ops = operations.log.all(BuildCacheDisableServiceBuildOperationType)
-        ops.size() == 1
-        with(ops[0].descriptor.details, BuildCacheDisableServiceBuildOperationType.Details) {
-            message == "a non-recoverable error was encountered"
-            reason == NON_RECOVERABLE_ERROR
-        }
-    }
```

Comment:
- [ ] If there's an error and we disable the cache, does it still get reported after this change?